src/NuGet.Clients/PackageManagement.UI/Xamls/PackageRestoreBar.xaml.cs

```diff
@@ -13,6 +13,7 @@
 using NuGet.Packaging;
 using NuGet.ProjectManagement;
 using VsBrushes = Microsoft.VisualStudio.Shell.VsBrushes;
+using System.Windows.Media.Animation;
```

Comment:
nit: there is an option in VS to sort System namespaces first and most all of the code has been sorted to use that